meeting-notes/2022-05-14.md

```diff
@@ -0,0 +1,261 @@
+# 5/14/22
+
+@pcohen, @rntz
+
+## TODOs
+
+- [ ] Phil JSON:
+  - add cursor selection ranges to JSON output
+- [ ] rntz JSON:
+  - Add selections (including the anchor)
+  - Multiple cursors
+  - Buffer filename should be null if nonexistent
+  - camelCase
+
+@rntz sez: "maybe we should put all these files in a ~/.cursorless/ directory?"
+
+## Notes
+
+**what phil's fork of the cursorless vscode plugin does**
+
+1. writes hats to a file when there's a change to those hats
+
+**phil next steps**
+
+- Migrate "watch JB/Emacs editor state and tell VS Code to navigate to that same
+  location (thus updating the hats)"logic to cursorless fork
+  (file watcher)
+
+- Make new JetBrains plugin also publish standardized \~/.editor-state.json,
+  since it\'s current JSON is overly complicated for this use case
+
+- Export forked changes in knausj VSCode command server
+
+- Export changes from custom command server
+
+**rntz next steps**
+
+- ~~Install and get building (./build)~~
+
+- Get emacs reading \~/.vscode-hats.json and applying hats
```

Comment:
There's been lots of discussion in #emacs channel on Slack—guessing @rntz is already up to date on all that tho and has an approach in mind